test/verify/check-kdump

```diff
@@ -103,25 +109,57 @@ class TestKdump(MachineCase):
         self.enableKdump()
         self.rebootMachine()
         self.browser.wait_visible("#app")
-
-        # we should have the amount of memory reserved that we indicated
-        self.browser.wait_in_text("#app", "256 MiB")
-        # service should start up properly and the button should be on
-        self.browser.wait_in_text("#app", "Service is running")
-        self.browser.wait_in_text(onOffSelectorActive, "On")
+        self.enableLocalSsh()
+
+        if self.machine.image not in ["rhel-7-6-distropkg"]:
+            # generate a valid kdump config with ssh target
+            settingsLink = "a:contains('locally in /var/crash')"
+            self.browser.wait_present(settingsLink)
+            self.browser.wait_visible(settingsLink)
+            self.browser.click(settingsLink, True)
+            self.browser.wait_present("#kdump-settings-location")
+            self.browser.click("#kdump-settings-location")
+            self.browser.wait_present("a:contains('Remote over SSH')")
+            self.browser.click("a:contains('Remote over SSH')")
+            sshInput = "#kdump-settings-ssh-server"
+            self.browser.wait_present(sshInput)
+            self.browser.wait_visible(sshInput)
+            self._type_val(sshInput, "root@localhost")
+            sshKeyInput = "#kdump-settings-ssh-key"
+            self.browser.wait_present(sshKeyInput)
+            self.browser.wait_visible(sshKeyInput)
+            self._type_val(sshKeyInput, "/root/.ssh/id_rsa")
+            pathInput = "#kdump-settings-local-directory"
+            self.browser.wait_present(pathInput)
+            self.browser.wait_visible(pathInput)
+            self._type_val(pathInput, "/var/crash")
+            self.browser.click("button.btn-primary:contains('Apply')")
+            self.browser.wait_not_present(pathInput)
+            # we should have the amount of memory reserved that we indicated
```

Comment:
These bits should again work on rhel-7-6-distropkg.

Reply:
you mean the entire section testing the nfs and ssh targets?

Reply:
No, just the "amount of memory" bits. Not sure about the new targets. I know nfs kinda sorta worked, but it might not have been good enough to satisfy the test case.

Reply:
As nfs wasn't available in the select menu, I'm afraid not (unless a previous nfs configuration was present).

But I'll reinstate the amount of memory bit.